pika/adapters/twisted_connection.py

```diff
@@ -397,7 +310,7 @@ def _adapter_get_write_buffer_size(self):
         """Implement pure virtual
         :py:ref:meth:`pika.connection.Connection._adapter_emit_data()` method.
 
-        TODO: this method only belongs in SelectConnection, none others needs it
+        TODO: this method only belongs in SelectConnection, no others need it
```

Comment:
Hmm, I don't see an implementation of `_adapter_get_write_buffer_size` in `SelectConnection` ... @vitaly-krugl is that something that should be there?